src/Analyzer.Core.JsonRuleTests/TestRunner.cs

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Azure.Templates.Analyzer.Types;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Templates.Analyzer.Core.JsonRuleTests
+{
+    [TestClass]
+    public class TestRunner
+    {
+        Dictionary<string, IEnumerable<IEvaluation>> templateEvaluations = new();
```

Comment:
Can we use the full constructor syntax? I personally think it's a little confusing understanding `new()` Thoughts @VeraBE

Reply:
I like `var templateEvaluations = new  Dictionary<string, IEnumerable<IEvaluation>>()` better too

Reply:
```suggestion
        var templateEvaluations = new Dictionary<string, IEnumerable<IEvaluation>>()
```

Reply:
(I wanted to test this new suggestion feature 😂)

Reply:
Two things:
- the `new()` syntax has been used in other places too.  Do we need to agree on a convention when it's ok to use?
  - Personally, I think it will be easy to get used to.  It might be confusing the first time seeing it, but you only need to gain understanding once.
- `var` can't be used for class members - either the full `Dictionary<...>` type needs to be written out twice, or we use `new()`.

Reply:
I didn't notice it was a class member! Personally, I don't think we need a convention for that, but I like using var every time it's possible, to focus on the variable name